snobal_1d/station_pipeline.py

```diff
@@ -0,0 +1,257 @@
+    # day branch
+    elif sw_net >= 10:   
+        if RH_col < 60:
+            # clear branch
+            x = clear['c1'] + clear['c2'] * RH_col + clear['c3'] * SB_CONST * air_t_col_K ** 4
+            lw_in_est.append(x)
+        elif RH_col >= 60:
+            # cloudy branch
+            x = cloudy['c1'] + cloudy['c2'] * RH_col + cloudy['c3'] * SB_CONST * air_t_col_K ** 4
+            lw_in_est.append(x)
+    
+    return lw_in_est
+
+
+def estimate_lw(df, lw_col, temp_col, threshold):
+    """
+    Simplified (collapsed) longwave parameterization from Skiles et al. 2012. 
+    
+    :param df: dataframe with input data
+    :param lw_col: column with incoming longwave radiation
+    :param threshold: theshold incoming longwave (in w/m^2) below which the correction is applied. We assume snow is occluding
+                      the sensor when it is below this theshold.
+    :param precip_col: column with binned precipitation data. 
+    :return: estimate of lw_in during precip, based on air t, in new col "incoming_radiation_lw_corr".
+    """
+    # create new col for corrected data
+    df['incoming_radiation_lw_corr'] = np.nan
+    # convert air temp from C to K
+    df['air_temp_K'] = df[temp_col].apply(lambda x: x + 273.15)
+    df.loc[(df[lw_col] < threshold) & (df['fraction'] != 0), 'incoming_radiation_lw_corr'] = 0.963*SB_CONST*df['air_temp_K']**4
+    return df
+
+    
+def replace_sw_in(df, default_sw_in, alt_sw_in, precip_col):
+    """replace sw_in at Atwater with sw_in from Reynolds peak when Atwater radiometer is occluded."""
+    df.loc[(df[default_sw_in] < df[alt_sw_in]) & (df[precip_col] != 0), 'corr_sw_in'] = df[alt_sw_in]
+    return df
+
+def snow_density_fraction(df, air_t_col):
+    """
+    Estimate density and fraction of snow based on air temp. Translated from IDL code 
+    written by TH Painter, and provided by S Skiles.
+    
+    :param df: pandas df to pull temp col from
+    :param air_t_col: air temp col (in deg C)
+    :return: df with new cols "fraction" and "density"
+    """
+    df['fraction'] = df[air_t_col].copy()
+    df['density'] = df[air_t_col].copy()
+    
+    def transform_fraction(x):
+        #if x != np.nan:
+            if x > 0.5:
+                x = 0.00
+                return x
+        
+            elif x < -5.0:
+                x = 1.00
+                return x
+
+            elif x > -5.0 and x < -3.0:
+                x = 1.00
+                return x
+                
+            elif x > -3.0 and x < -1.0:
+                x = 1.00
+                return x
+            elif x > -1.0 and x < -0.5:
+                x = 1.00
+                return x
+                
+            elif x > -0.5 and x < 0.0:
+                x = 0.75
+                return x
+                
+            elif x > 0.0 and x < 0.5:
+                x = 0.25
+                return x
+                           
+    def transform_density(x):
+        #if x != np.nan:
```

Comment:
I would remove commented out code completely to make it easier to read.

Reply:
Commented out code is removed. I've also added "Icon" to my `.gitignore`. What is the best way to remove it from this repo. Can I just delete it?

Reply:
For cases like the icon, where the file size is 0 bytes, I would say yes. Just delete them and commit that change.